File: platform-mobile/src/main/java/ru/pocketbyte/locolaser/platform/mobile/IosObjectiveCPlatformConfig.java
package ru.pocketbyte.locolaser.platform.mobile;

import ru.pocketbyte.locolaser.platform.mobile.resource.IosObjectiveCResources;
import ru.pocketbyte.locolaser.resource.PlatformResources;

public class IosObjectiveCPlatformConfig extends IosBaseClassPlatformConfig {

    public static final String TYPE = "ios_objc";

    @Override
    public String getType() {
        return TYPE;
    }

    @Override
    public PlatformResources getResources() {
        return new IosObjectiveCResources(getResourcesDir(), getResourceName(), getTableName());
    }

}

File: platform-mobile/src/main/java/ru/pocketbyte/locolaser/platform/mobile/IosObjectiveCPlatformConfig.kt
package ru.pocketbyte.locolaser.platform.mobile

import ru.pocketbyte.locolaser.platform.mobile.resource.IosObjectiveCResources

class IosObjectiveCPlatformConfig : IosBaseClassPlatformConfig() {

    companion object {
        const val TYPE = "ios_objc"
    }

    override val type = TYPE
    override val resources
        get() = IosObjectiveCResources(resourcesDir!!, resourceName!!, tableName, filter)

}
